wpilibc/src/main/native/cpp/logging/CSVLogFile.cpp

```diff
@@ -0,0 +1,61 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2019 FIRST. All Rights Reserved.                             */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+#include "frc/logging/CSVLogFile.h"
+
+#include <chrono>
+#include <iostream>
+
+using namespace frc;
+
+CSVLogFile::CSVLogFile(std::string filePrefix)
+    : LogFile(filePrefix, "csv"),
+      m_cells(),
+      m_timestampCell("Timestamp (ms)"),
```

Comment:
Initialize these in the header like Java when possible. m_cells() is redundant and can be removed. The only thing you need in the initialization list here is the delegation to the base class.

Reply:
What does the "header" means ? The header file ?

Reply:
As an example, you can definitely initialise `m_active` in the .h file where you declare the member variable. You can take a look at the other classes for some concrete examples.

Reply:
Ok I did it with `m_active` but how do I do it with objects ?